### PR TITLE
Wait time for restaurants

### PR DESCRIPTION
**Pull request by eman1408**
Added Rufus Scheduler to run wait_time method every 5 seconds 
Fixed place in queue, people waiting in the queue on home page

### CODE REVIEW

File: app/controllers/restaurants_controller.rb [hunk 1]
```diff
@@ -12,20 +13,24 @@ def create
...
   end
 
-
   def show
     @restaurant = Restaurant.find(params[:id])
+    # @restaurant.update(total_wait_time: wait_time)
+    job_id =
+      Rufus::Scheduler.singleton.every '5s' do
+        Rails.logger.info "TEST ME time flies, it's now #{Time.now}"
+        wait_time
```
Comment (TheDemonGuard) [inline app/controllers/restaurants_controller.rb:30]: Here every time the "show" action is called, new schedule is going to get create.
Adding more schedule to the existing one.
Maybe we can find a way to set the schedule only once.

**Review by TheDemonGuard — APPROVED** [review]
Maybe we can find the way to set scheduler only once in the future.
At this time we can use this for the upcoming Pitch.

File: app/controllers/restaurants_controller.rb (hunk 1 above)
Comment (eman1408) [reply app/controllers/restaurants_controller.rb:30]: yes I agree we can use it temporarily until we figure out how to call it once. Perhaps in the application.rb file or something like that when it starts up

Comment (eman1408) [reply app/controllers/restaurants_controller.rb:30]: or perhaps we can stop and start the method when we call it so if its running stop then start again

**Pull request merged**